didcomm/message.py

```diff
@@ -102,7 +102,7 @@ def as_dict(self) -> dict:
             d["from"] = d["frm"]
             del d["frm"]
 
-        d["typ"] = DIDCommMessageTypes.PLAINTEXT.value
+        d["typ"] = DIDCommMessageTypes.PLAINTEXT.value # this is not part of the official specification
```

Comment:
If this is optional, I would rather leave it here. Not require it, but always put.